website/docs/reference/warehouse-profiles/rockset-profile.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Rockset Profile"
+---
+
+:::info Vendor-supported plugin
+
+Certain core functionality may vary. If you would like to report a bug, request a feature, or contribute, you can check out the linked repository and open an issue.
+
+:::
+
+## Overview of dbt-rockset
+
+**Maintained by:** Rockset, Inc.      
+**Source:** https://github.com/rockset/dbt-rockset
```

Comment:
Could you add two whitespaces behind `**Source:** https://github.com/rockset/dbt-rockset` ? Markdown is funny with lists.

I also spent some time standardizing the adapter profiles so could you add a new section about dbt Cloud being not supported (as seen in this [Materialize profile](https://docs.getdbt.com/reference/warehouse-profiles/materialize-profile)) and link the github repo rather than the URL completely?

Reply:
We can go ahead and create the slack channel when it's ready